server/mergin/sync/public_api_v2.yaml

```diff
@@ -0,0 +1,67 @@
+openapi: 3.0.0
+info:
+  description: Mergin API to synchronize your GIS data.
+  version: "0.6"
+  title: Mergin API
+servers:
+  - url: /v2
```

Comment:
for this to be "public" endpoint you need to add it to CSRF exceptions https://github.com/MerginMaps/mergin/blob/master/server/mergin/app.py#L209